### DISCUSSION TITLE
Default of 3840 sec window for PCAP extraction?

### DISCUSSION THREAD

**Discussion by hbeall1973** [Security-Onion-Solutions/securityonion#5922] [Unsupported Versions]
New to SO, but I'm hoping this is a simple answer...  I'm running SO 2.3.70

I'm testing out SO in the lab, and I have a question about the amount of time that is selected when extracting PCAPs through the "Hunt" interface.  When you select an entry, goto PCAP, the timeframe in the query is always 3840 "wide" (64 min).  So from the time of the incident, it looks like it's going 1920 seconds (32 min) before and after it.  

![image](https://user-images.githubusercontent.com/63920285/137581673-dc974b0b-ab4a-41f5-a4b2-d13a15e05c5d.png)

The problem that I'm seeing is that if I select an entry that is 5 min old, it's not going to complete for another 27'ish min (5+27=32).  I see the job as being incomplete, but will eventually complete after this 32 min additional time is passed.

![image](https://user-images.githubusercontent.com/63920285/137582499-8d0068ee-3841-4491-943e-0b559414d891.png)

Is there any way that I can modify this 32 min (1920 sec) time window value?

**Comment by jertel** [comment]
First, the filter end time being in the future does not prevent a job from completing. 
Second, yes, it's possible to change the window time value by customizing a salt file, but this customization will be lost on future upgrades. That said, this shouldn't be necessary per my first response so I'll hold off for now on giving details on that customization, to avoid confusing other readers.

Double check that your sensor node and manager nodes are correctly time synced. Then review your sensor's `/opt/so/log/sensoroni/sensoroni.log` file to determine why it is taking longer than usual to complete jobs.

**Reply by jertel** [reply]
Here's an example of a job I recently ran. Notice how the queue date and completion date are both well before the filter end date.

![image](https://user-images.githubusercontent.com/16639611/137720100-d013c828-1857-4401-82b3-b62e6636d2d7.png)
